### ISSUE TITLE
Electron exited with code 2147483651.

### ISSUE THREAD

**Issue by Irasil** [electron/fiddle#1462]
After the last update 0.34.1 I always get this exit code. The workaround is to uninstall and reinstall Fiddle. Ignore the update that happens in the background and never close Fiddle. :)

**Comment by dsanders11** [comment]
More information would be helpful here. What operating system are you running into this on?

**Comment by Irasil** [comment]
Sorry, maybe it's just me, today is the first day I'm using Electron Fiddle 

Here are my OS Information:

- Edition:  Windows 11 Pro
- Version:	        22H2
- OS build:	        22621.2283
- Experience:	Windows Feature Experience Pack 1000.22662.1000.0

When I run the default project under 0.34.0
![34 0](https://github.com/electron/fiddle/assets/91846030/e28af756-7897-48dd-a070-fa8b5054df52)

After a while I get the message that there was an update and I should restart:
![restart](https://github.com/electron/fiddle/assets/91846030/096ce65a-83ef-49db-b922-5dfbc3293ea7)

And after restarting the app
![34 1](https://github.com/electron/fiddle/assets/91846030/c21c90d7-f994-461c-9687-6462d045f286)

The installer I use is "electron-fiddle-0.34.0-win32-x64-setup.exe" from [electronjs.org](https://www.electronjs.org/fiddle)

Do you need any other information?

**Comment by dsanders11** [comment]
Thanks for the extra info. It also reproduces on Windows 10.

**Comment by brhenrique** [comment]
Same on Windows 11

```
[21288:0921/104336.580:FATAL:chrome_paths.cc(158)] Check failed: false.
Backtrace:
base::debug::CollectStackTrace [0x00007FF73BFD3812+18] (o:\base\debug\stack_trace_win.cc:329)
base::debug::StackTrace::StackTrace [0x00007FF73BFC9142+18] (o:\base\debug\stack_trace.cc:218)
logging::LogMessage::~LogMessage [0x00007FF73BF15A68+184] (o:\base\logging.cc:730)
logging::`anonymous namespace'::NotReachedLogMessage::~NotReachedLogMessage [0x00007FF73BEFDC90+336] (o:\base\check.cc:60)
chrome::PathProvider [0x00007FF73CBFEA1F+271] (o:\chrome\common\chrome_paths.cc:158)
base::PathService::Get [0x00007FF73BFCEBCF+847] (o:\base\path_service.cc:213)
electron::`anonymous namespace'::ElectronPathProvider [0x00007FF737C19FE9+393] (o:\electron\shell\app\electron_main_delegate.cc:147)
base::PathService::Get [0x00007FF73BFCEBCF+847] (o:\base\path_service.cc:213)
BrowserProcessImpl::PostEarlyInitialization [0x00007FF737D21F87+229] (o:\electron\shell\browser\browser_process_impl.cc:111)
electron::ElectronBrowserMainParts::PostEarlyInitialization [0x00007FF737D34ED0+576] (o:\electron\shell\browser\electron_browser_main_parts.cc:312)
content::BrowserMainLoop::EarlyInitialization [0x00007FF73ADC6B5D+1217] (o:\content\browser\browser_main_loop.cc:628)
content::BrowserMainRunnerImpl::Initialize [0x00007FF73ADCB5EF+495] (o:\content\browser\browser_main_runner_impl.cc:117)
content::BrowserMain [0x00007FF73ADC5C7A+142] (o:\content\browser\browser_main.cc:28)
content::RunBrowserProcessMain [0x00007FF737F3B30B+347] (o:\content\app\content_main_runner_impl.cc:720)
content::ContentMainRunnerImpl::RunBrowser [0x00007FF737F3D5E7+1237] (o:\content\app\content_main_runner_impl.cc:1283)
content::ContentMainRunnerImpl::Run [0x00007FF737F3D014+1268] (o:\content\app\content_main_runner_impl.cc:1141)
content::RunContentProcess [0x00007FF737F37EBB+1389] (o:\content\app\content_main.cc:328)
content::ContentMain [0x00007FF737F37FF6+111] (o:\content\app\content_main.cc:345)
wWinMain [0x00007FF737C114DF+915] (o:\electron\shell\app\electron_main_win.cc:243)
__scrt_common_main_seh [0x00007FF74170A592+262] (D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288)
BaseThreadInitThunk [0x00007FF89A4E257D+29]
RtlUserThreadStart [0x00007FF89C14AA68+40]
Crash keys:
"platform" = "win32"
"process_type" = "browser"

Electron exited with code 2147483651.
```

**Comment by darkless47** [comment]
Edition	Windows 10 Home
Version	22H2
OS build	19045.3448
Experience	Windows Feature Experience Pack 1000.19044.1000.0

Installer: electron-fiddle-0.34.1-win32-x64-setup.exe from electronjs.org

Hello world example from electronjs.org
![fiddle-issue](https://github.com/electron/fiddle/assets/44588509/71f2ae73-8a85-42cd-8b41-e64f49bca8df)

**Comment by Irasil** [comment]
The new update works without any problems on my Mac
Mac mini
macOS: Ventura 13.5.2
Chip Apple M1

![Screenshot 2023-09-21 155438](https://github.com/electron/fiddle/assets/91846030/b51aecee-df4f-43a1-93a4-483c8e81e94d)

**Comment by brhenrique** [comment]
@codebytere it's a debug build of Electron 24.0.0

The error itself comes from this:

```
[41408:0921/112950.423:ERROR:base_paths_win.cc(141)] [DEBUG] SHGetFolderPath -2147024893
```

For some reason, in this new version of Electron fiddle, the Electron targets try to get the path of `base::DIR_LOCAL_APP_DATA` and fails. But using Electron fiddle 0.34.0, the Electron build I tested did not try to get the path of local app data.

**Comment by dsanders11** [comment]
Looks like this was caused by a bad build, we've released v0.34.2 and I've tested it on Windows and it should be fixed. 👍
